### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by syc728**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,9 +34,13 @@ python sum.py -i sample.mp4 -s subtitle.srt
 ```
 To download a YouTube video along with its subtitles from url:
```
Comment (AdiChat) [inline README.md]: This should be "To summarize a YouTube video from its url:"

File: README.md [hunk 2]
```diff
@@ -34,9 +34,13 @@ python sum.py -i sample.mp4 -s subtitle.srt
...
 ```python
-python sum.py -u "url"
+python sum.py -u <url>
```
Comment (AdiChat) [inline README.md]: Need to mention that the downloaded files will remain in this case 👍

Comment (syc728) [reply README.md]: Only with -k, the downloaded files will be kept.

**Comment by syc728** [comment]
@AdiChat Please review the PR with updated README.md

**Comment by AdiChat** [comment]
Looks good 👍 

Hope you are enjoying your journey with **OpenGenus**

**Pull request merged**